python/federatedml/secret_sharing_sum/secret_sharing_sum_guest.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+#  Copyright 2019 The FATE Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from federatedml.util import LOGGER
+from federatedml.transfer_variable.transfer_class import secret_sharing_sum_transfer_variable
+from federatedml.param.secure_sharing_sum_param import SecureSharingSumParam
+from federatedml.secret_sharing_sum.base_secret_sharing_sum import BaseSecretSharingSum
+
+
+class SecretSharingSumGuest(BaseSecretSharingSum):
+    def __init__(self):
+        super(SecretSharingSumGuest, self).__init__()
+        self.transfer_inst = secret_sharing_sum_transfer_variable.SecretSharingSumTransferVariables()
+
+    def _init_model(self, model_param: SecureSharingSumParam):
+        self.sum_cols = model_param.sum_cols
+
+    def _init_data(self, data_inst):
+        self.host_count = len(self.component_properties.host_party_idlist)
+        self.vss.set_share_amount(self.host_count+1)
```

Comment:
convoluted way of counting share amount, consider use *(x+1) in vss computation, and use the real value of needed share_amount.